backend/src/main/java/com/bakdata/conquery/io/jackson/Initializing.java

```diff
@@ -0,0 +1,26 @@
+package com.bakdata.conquery.io.jackson;
+
+import com.fasterxml.jackson.databind.util.StdConverter;
+
+/**
+ * Interface for class instances that need initialization after deserialization and value injection.
+ * Let the class implement this interface and annotate the class with:
+ * <pre>
+ *  {@code
+ *  @JsonDeserialize(converter = Initializing.Converter.class )
+ *  }
+ * </pre>
+ * @param <T>
+ */
+public interface Initializing<T extends Initializing<T>> {
+
+	T init();
+
+	class Converter<T extends Initializing<T>> extends StdConverter<T, T> {
+
+		@Override
+		public T convert(T value) {
+			return value.init();
+		}
+	}
```

Comment:
wäre mir glaube lieber - und einfacher -  den Code erstmal nur bei init zu belassen; und dann eben 
```suggestion
	class Converter<T extends Initializing<T>> extends StdConverter<T, T> {

		@Override
		public T convert(T value) {
			value.init();
			return value;
		}
	}
```

zu machen

Reply:
Mit init als extra Aufruf beißt es sich beim Caching, da man dann beim laden eines Objektes noch daran denken muss es zu initializieren